src/main/java/com/synopsys/integration/detect/lifecycle/autonomous/AutonomousManager.java

```diff
@@ -34,9 +38,10 @@ public class AutonomousManager {
     private String blackDuckScanMode;
     private final DetectPropertyConfiguration detectConfiguration;
     private SortedMap<String, String> userProvidedProperties = new TreeMap<>();
-    private SortedMap<String, String> globalProperties = new TreeMap<>();
-    private SortedMap<String, String> detectorSharedProperties = new TreeMap<>();
-    private static final List<String> propertiesNotAutonomous = Arrays.asList("blackduck.api.token", "detect.diagnostic", "detect.source.path", "detect.tools");
+    private SortedMap<String, String> allProperties = new TreeMap<>();
+    private List<String> ranScanTypes = new ArrayList<>();
+    private List<String> ranDetectorTypes = new ArrayList<>();
+    private static final List<String> propertiesNotAutonomous = Arrays.asList("blackduck.api.token", "detect.diagnostic", "detect.source.path", "detect.tools", "blackduck.proxy.password");
```

Comment:
Re detect.tools being included in scan-settings file -- is this being handled in the upcoming PR?

Reply:
Yes, will be handled in the next PR.